src/test/java/teammates/test/cases/ui/pagedata/StudentCommentsPageDataTest.java

```diff
@@ -0,0 +1,231 @@
+        CourseRoster roster = new CourseRoster(students, instructors);
+        String studentEmail = student1.email;
+        
+        // create a single feedbackResultBundles for init
+        Map<String, FeedbackSessionResultsBundle> feedbackResultBundles = 
+                new HashMap<String, FeedbackSessionResultsBundle>();
+        FeedbackSessionAttributes session = dataBundle.feedbackSessions.get("session1InCourse1");
+        FeedbackResponseAttributes response = dataBundle.feedbackResponses.get("response1ForQ1S1C1"); 
+        response.setId("1");
+        List<FeedbackResponseAttributes> responses = Arrays.asList(response);
+        Map<String, FeedbackQuestionAttributes> relevantQuestions = new HashMap<String, FeedbackQuestionAttributes>();
+        FeedbackQuestionAttributes question = dataBundle.feedbackQuestions.get("qn1InSession1InCourse1");
+        question.setId("1");
+        relevantQuestions.put(question.getId(), question);
+        Map<String, String> emailNameTable = new HashMap<String, String>();
+        Map<String, String> emailLastNameTable = new HashMap<String, String>();
+        Map<String, String> emailTeamNameTable = new HashMap<String, String>();
+        Map<String, boolean[]> visibilityTable = new HashMap<String, boolean[]>();
+        Map<String, List<FeedbackResponseCommentAttributes>> responseComments =
+                new HashMap<String, List<FeedbackResponseCommentAttributes>>();
+        emailNameTable.put(student1.email, student1.name);
+        emailLastNameTable.put(student1.email, student1.lastName);
+        emailTeamNameTable.put(student1.email, student1.team);
+        boolean[] visibility = {true, true};
+        visibilityTable.put(response.getId(), visibility);
+        FeedbackSessionResponseStatus responseStatus = new FeedbackSessionResponseStatus();
+        responseStatus.emailNameTable.put(student1.email, student1.name);
+        responseStatus.emailTeamNameTable.put(student1.email, student1.team);
+        FeedbackResponseCommentAttributes responseComment = 
+                dataBundle.feedbackResponseComments.get("comment1FromT1C1ToR1Q1S1C1");
+        List<FeedbackResponseCommentAttributes> responseCommentsList = Arrays.asList(responseComment);
+        responseComments.put(response.getId(), responseCommentsList);
+        boolean isComplete = true;
+        FeedbackSessionResultsBundle results =
+                new FeedbackSessionResultsBundle(
+                        session, responses, relevantQuestions,
+                        emailNameTable, emailLastNameTable, emailTeamNameTable,
+                        visibilityTable, responseStatus, roster, responseComments, isComplete);
+        feedbackResultBundles.put(session.feedbackSessionName, results);
+        
+        data.init(courseId, courseName, coursePaginationList, comments, roster, studentEmail, feedbackResultBundles);
+        
+        /**************************** Assertions ****************************/
```

Comment:
Let's not call these (JUnit comparisons) assertions, to avoid confusion with java asserts.